### PR TITLE
Vanilla upgrade 2.0.1

### PR DESCRIPTION
**Pull request by kwm14**
## Done
- An updated version of the site to v2.0.1
- Updated components, classes, and utilities based on release notes [here](https://github.com/canonical-web-and-design/vanilla-framework/releases/tag/2.0.1)
- Visual and layout updates 😉 

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8009/](http://0.0.0.0:8009/)
4. Or view demo link below 👍 

## Fixes
- New team members - https://github.com/canonical-web-and-design/cloud-init.io/issues/104

## Screenshot
cloud init   The standard for customising cloud instances (2)

### CODE REVIEW

**Comment by pmahnke** [comment]
@kwm14 looks great!

File: yarn.lock [hunk 1]
```diff
@@ -2994,9 +2994,9 @@ validate-npm-package-license@^3.0.1:
     spdx-correct "~1.0.0"
     spdx-expression-parse "~1.0.0"
 
-vanilla-framework@1.8.1:
-  version "1.8.1"
-  resolved "https://registry.yarnpkg.com/vanilla-framework/-/vanilla-framework-1.8.1.tgz#0d7f5885cdbd1355a9f9d42d7596f9a1258b27ea"
+vanilla-framework@2.0.0-alpha:
```
Comment (deadlight) [inline yarn.lock]: This still has the alpha. ./run the project and then commit the automatic changes that are made to the lockfile

File: _includes/footer.html [hunk 2]
```diff
@@ -1,29 +1,12 @@
...
-            </a>
-          </li>
-          <li class="p-footer__item">
-            <a class="p-footer__link"
-              aria-label="External link to report a bug on this site"
-              href="https://github.com/ubuntudesign/cloud-init/issues/new">
-              Report a bug on this site
-            </a>
-          </li>
+<div class="p-strip is-shallow">
```
Comment (deadlight) [inline _includes/footer.html]: Can you update this file to two-space indenting, please?

Comment (kwm14) [reply _includes/footer.html]: Updated indentation 😄

File: yarn.lock (hunk 1 above)
Comment (kwm14) [reply yarn.lock]: Our `./run` the project based on updates. Hopefully, this has fixed the lockfile 🤔

Comment (deadlight) [reply yarn.lock]: @kwm14 I forgot to tell you to `./run clean` first, sorry

**Pull request merged**